# ICE


Reference

[1] Peng W, Yi S, Dai W, et al. Identifying and ranking potential cancer drivers using representation learning on attributed network[J]. Methods, 2021, 192: 13-24. https://github.com/weiba/RLAG 

[2] Song J, Peng W, Wang F. An Entropy-based method for identifying mutual exclusive driver genes in cancer[J]. IEEE/ACM transactions on computational biology and bioinformatics, 2019, 17(3): 758-768. https://github.com/weiba/EntroRank

[3] Horn H, Lawrence M S, Chouinard C R, et al. NetSig: network-based discovery from cancer genomes[J]. Nature methods, 2018, 15(1): 61-66.

[4] Reyna M A, Leiserson M D M, Raphael B J. Hierarchical HotNet: identifying hierarchies of altered subnetworks[J]. Bioinformatics, 2018, 34(17): i972-i980. https://github.com/raphael-group/hierarchical-hotnet

[5] Kumar R D, Swamidass S J, Bose R. Unsupervised detection of cancer driver mutations with parsimony-guided learning[J]. Nature genetics, 2016, 48(10): 1288-1294.https://github.com/bose-lab/ParsSNP

[6] Cho A, Shim J E, Kim E, et al. MUFFINN: cancer gene discovery via network analysis of somatic mutation data[J]. Genome biology, 2016, 17(1): 1-16. https://github.com/netbiolab/MUFFINN

[7] Shi K, Gao L, Wang B. Discovering potential cancer driver genes by an integrated network-based approach[J]. Molecular BioSystems, 2016, 12(9): 2921-2931.

[8] Hou J P, Ma J. DawnRank: discovering personalized driver genes in cancer[J]. Genome medicine, 2014, 6(7): 1-16.

[9] Leiserson M D M, Vandin F, Wu H T, et al. Pan-cancer network analysis identifies combinations of rare somatic mutations across pathways and protein complexes[J]. Nature genetics, 2015, 47(2): 106-114. https://github.com/raphael-group/hotnet2
